extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/config/VaultServerRuntimeConfig.java

```diff
@@ -0,0 +1,223 @@
+package io.quarkus.vault.runtime.config;
+
+import static io.quarkus.vault.runtime.LogConfidentialityLevel.HIGH;
+import static io.quarkus.vault.runtime.LogConfidentialityLevel.LOW;
+
+import java.net.URL;
+import java.time.Duration;
+import java.util.Map;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.quarkus.vault.runtime.LogConfidentialityLevel;
+
+@ConfigRoot(name = "vault", phase = ConfigPhase.RUN_TIME)
+public class VaultServerRuntimeConfig {
```

Comment:
Just call it `VaultRuntimeConfig` and use convention over configuration? You should be able to release the name.

Reply:
I renamed it, but was not able to drop the name:
```
2019-10-05 01:32:31,399 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.vault.url" provided
```